rtrlib/lib/alloc_utils.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of RTRlib.
+ *
+ * This file is subject to the terms and conditions of the MIT license.
+ * See the file LICENSE in the top level directory for more details.
+ *
+ * Website: http://rtrlib.realmv6.org/
+ */
+
+#include <stdlib.h>
+
+#include "alloc_utils.h"
+
+void *(*MALLOC_PTR)(size_t size) = malloc;
+void *(*REALLOC_PTR)(void *ptr, size_t size) = realloc;
+void (*FREE_PTR)(void *ptr) = free;
```

Comment:
Now that I think about it since they are not constants a lowercase name would fit and of course we do not want your eyes to hurd ;)